libsolidity/lsp/LanguageServer.h

```diff
@@ -0,0 +1,198 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+#pragma once
+
+#include <libsolidity/formal/ModelCheckerSettings.h>
+#include <libsolidity/interface/CompilerStack.h>
+#include <libsolidity/interface/FileReader.h>
+#include <libsolidity/interface/ImportRemapper.h>
+#include <libsolidity/lsp/LSPTypes.h>
+#include <libsolidity/lsp/ReferenceCollector.h>
+#include <libsolidity/lsp/Transport.h>
+
+#include <libsolidity/ast/AST.h>
+
+#include <liblangutil/SourceReferenceExtractor.h>
+
+#include <fmt/format.h>
+
+#include <json/value.h>
+
+#include <boost/filesystem/path.hpp>
+
+#include <functional>
+#include <iostream>
+#include <map>
+#include <memory>
+#include <optional>
+#include <ostream>
+#include <set>
+#include <string>
+#include <string_view>
+#include <unordered_map>
+#include <variant>
+#include <vector>
+
+namespace solidity::frontend {
```

Comment:
`{` goes to the start of the next line (except in expression context). Please correct all these cases.